test/Microsoft.Sbom.Tool.Tests/IntegrationTests.cs

```diff
@@ -202,7 +202,7 @@ public void E2E_Generate_WithBadManifestInfo_ReturnsNonZeroExitCode()
         var testFolderPath = CreateTestFolder();
         var arguments = $"generate -ps IntegrationTests -pn IntegrationTests -pv 1.2.3 -m \"{testFolderPath}\" -b \"{testDropDirectory}\" -bc \"{GetSolutionFolderPath()}\" -mi randomName:randomVersion";
         var (stdout, stderr, exitCode) = LaunchAndCaptureOutput(arguments);
-        Assert.AreEqual("Please provide a valid value for the ManifestInfo (-mi) parameter. Supported values include: SPDX:2.2, SPDX:3.0. The values are case-insensitive.\r\n", stderr);
+        Assert.AreEqual("The value 'randomName:randomVersion' contains no values supported by the ManifestInfo (-mi) parameter. Please provide supported values. Supported values include: SPDX:2.2, SPDX:3.0. The values are case-insensitive.\r\n", stderr);
```

Comment:
We should probably remove this part since it doesn't include supported values that might come from dependency injection

Reply:
These tests only support the types that we define in sbom-tool. We'll update it if/when we change the built-in types